run/test/specs/input_validations/onboarding_long_name.spec.ts

```diff
@@ -12,6 +12,10 @@ const tooLongName = 'One morning, when Gregor Samsa woke from troubled dreams, h
 const expectedError = localize('displayNameErrorDescriptionShorter').strip().toString();
 
 async function onboardingLongName(platform:SupportedPlatformsType) {
+  // this check is to avoid false positives 
```

Comment:
I meant specifically to add this check right above the inputText function where you use the value. Then if the value is incorrect it fails where the input occurs